ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXDualBlackflyProjection.java

```diff
@@ -81,17 +81,17 @@ public RDXDualBlackflyProjection(HumanoidReferenceFrames currentRobotFrames)
    {
       this.baseUI = RDXBaseUI.getInstance();
       this.robotZUpFrame = currentRobotFrames.getMidFootZUpGroundFrame();
-      robotCameraFrames.set(currentRobotFrames::getSituationalAwarenessCameraFrame);
+      robotCameraFrames.set(currentRobotFrames::getStereoCameraFrame);
 
```

Comment:
renamed to stereo camera, as it's a better description of what those cameras are. Especially considering that we are dropping the blackflies